developer-certificate-of-origin.txt

```diff
@@ -0,0 +1,35 @@
+Developer Certificate of Origin
```

Comment:
I think this document should be linked in the CONTRIBUTING, the README or the PR template.

Reply:
In the PR Template the CONTRIBUTING is mentioned and there the DCO is linked
![image](https://user-images.githubusercontent.com/3978662/99955348-7feb6b80-2d84-11eb-8337-b1f3b0183e78.png)